### DISCUSSION TITLE
Media Folders Permissions like Content Tree

### DISCUSSION THREAD

**Discussion by biapar** [umbraco/Umbraco-CMS#13503] [Features and ideas]
Media Folders Permissions like Content Tree

**Comment by bergmania** [comment]
Hi @biapar 

Can you elaborate a bit about what you need to what use cases..

We already have start nodes for media, which solves most use cases..

I guess you request permissions like Create, Delete, Move, Sort?

**Reply by biapar** [reply]
Yes. Also to read. And permission to "download"/REST with the link/URL.

**Reply by bergmania** [reply]
I dont understand what you mean by "/REST".

All files are currently available online, if you know the url. This is intended, as it would require time-expensive lookups otherwise, and prevent most caching.

So do you mean, get the URL? The would already have this, if they could browse the media item in the backoffice.

**Reply by biapar** [reply]
I mean to configure some permission on media folder or media file and then you cannot use the media or read it also by url. For example: an image has deny read permission for "guest". Then is not possible to visualize media also by url or into the backend.
